test/parallel/test-performance-measure-detail.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const util = require('util');
+const { performance, PerformanceObserver } = require('perf_hooks');
+
+const perfObserver = new PerformanceObserver((items) => {
+  items.getEntries().forEach((entry) => {
```

Comment:
Nit: There has been multiple attempts at `code-and-learn` events to convert `forEach` calls with `for of` (Ref: https://github.com/nodejs/node/pulls?q=is%3Apr+forEach+is%3Aclosed+label%3Acode-and-learn). I never asked why, or questioned but maybe we should convert this to `for of`?

Reply:
Done ! Thanks !